src/vehicle.h

```diff
@@ -242,22 +242,22 @@ class turret_data
          * and performs any other actions that must be done before firing a turret.
          * @param p the player that is firing the gun, subject to recoil adjustment.
          */
-        void prepare_fire( player &p );
+        void prepare_fire( Character &p );
```

Comment:
clang-tidy gets sad that the param `Character &p` is not `Character &who` lmao

```suggestion
        void prepare_fire( Character &who );
```